
    function f() {
        const u =0;
        while(u <10){
            let zufallszahl = Math.round(Math.random() *10 +1);
            if(zufallszahl > 30){
                throw new Error ("Fehler!");
                return fertigstellung;
            }
            u++;
        }
    }
    try {error}

     catch (error) {console.log(error)}